ferrocene/doc/release-notes/src/24.05.0.rst

```diff
@@ -42,7 +42,6 @@ shipped as a preview.
   - ``clippy``
   - ``rustfmt``
   - ``rust-analyzer``
-  - ``rust-demangler``
```

Comment:
would not remove it from here, as it's part of that release

Reply:
oops you are correct.